pylint/checkers/utils.py

```diff
@@ -231,6 +232,8 @@
 )
 
 T_Node = TypeVar("T_Node", bound=nodes.NodeNG)
+CheckerT = TypeVar("CheckerT", bound=BaseChecker)
+AstCallback = Callable[[CheckerT, T_Node], None]
```

Comment:
You'll need to import `Callable` from `typing` I think. Whenever it is used in runtime it needs to be from `typing` (until 3.9).

Reply:
It is already imported from ``typing``. 
I could solve it locally with putting it inside a ``if TYPE_CHECKING`` block. Let's see if the CI is happy with that.